lib/subscription.js

```diff
@@ -3,7 +3,7 @@ const Websocket = require('fastify-websocket')
 const SubscriptionConnection = require('./subscription-connection')
 const GRAPHQL_WS = 'graphql-ws'
 
-function handle (conn) {
+export function handle (conn) {
```

Comment:
This will not work outside of tap, you should use `module.export`. However I would prefer this to be tested by opening a websocket connection to a different route instead.